projects/media-viewer/src/lib/comments-summary/comments-summary.component.html

```diff
@@ -1,22 +1,32 @@
 <div id="modal-background" class="modal" (click)="onClose()">
   <div id="modal" class="modal-content govuk-width-container clearfix" (click)="$event.stopPropagation()">
-    <h2>{{ title }}</h2>
     <div id="comment-container" class="comment-container" #commentContainer>
+      <h2 class="govuk-heading-m">{{ title }}</h2>
       <table class="govuk-table">
-        <thead>
-          <tr class="govuk-table__header">
-            <th>User</th>
-            <th>Date</th>
-            <th>Comment</th>
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">User</th>
+            <th scope="col" class="govuk-table__header">Page</th>
+            <th scope="col" class="govuk-table__header">Date</th>
+            <th scope="col" class="govuk-table__header">Comment</th>
           </tr>
         </thead>
-        <tbody>
-          <tr *ngFor='let comment of comments; let i = index' [id]="'comment-row-'+i" class="govuk-table__cell">
-            <td><span title="{{comment.createdByDetails.email}}">{{comment.createdByDetails.forename}} {{comment.createdByDetails.surname}}</span></td>
-            <td>{{comment.createdDate | date : "hh:mm d MMMM yyyy" }}</td>
-            <td title="{{comment.content}}">
+        <tbody class="govuk-table__body">
+          <tr *ngFor='let comment of comments; let i = index' [id]="'comment-row-'+i" class="govuk-table__row">
+            <td class="govuk-table__cell">
+              <span title="{{comment.comment.createdByDetails.forename}} {{comment.comment.createdByDetails.surname}}">
+                {{comment.comment.createdByDetails.forename}} {{comment.comment.createdByDetails.surname}}
```

Comment:
would it make sense to use another variable name instead of comment, so that we don't end up with comment.comment